srcalc/src/main/webapp/js/enterVariables.js

```diff
@@ -1,79 +1,72 @@
 /* JavaScript code just for the enterVariables page */
 
-$(document).ready(function(){
+// We're about to replace the procedureSelectGroup with a jQuery UI
+// dialog. Insert a hidden input and a textual display as the target
+// of the user selection from the dialog.
+// Get the procedure hidden input, if it exists.
+function selectProcedure(cptCode, displayString) {
```

Comment:
Please move this function into initiProcedureSelect() to avoid polluting the global scope.